client/MessageForm.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import styles from './MessageForm.css';
+
+class MessageForm extends Component {
+    constructor(props) {
+        super(props);
```

Comment:
props are unnecessary in super ()  and in constructor when you doesn't change it in constructor